setup.py

```diff
@@ -8,7 +8,7 @@ def readme():
 
 setup(
     name="larq-zoo",
-    version="0.3.0",
+    version="0.4.0",
```

Comment:
I am not sure if this is right. Since this is breaking (in that the new zookeeper version is incompatible with the old one) I assume we go from 0.3.0 to 0.4.0.

Reply:
It looks like so far each release had either some new model or improved weights of already existing models in it. For my feeling this is more a 0.3.1 but I am also a bit uncertain about this.

Reply:
@koenhelwegen could you shine some light here?

Reply:
There is no need to change the version number when you change the code - it's fine if the master branch is non-compatible with the latest release. Releases are used to make a specific version of the code easily available (e.g. after a bug fix or when several changes are made). This is done by going to [releases](https://github.com/larq/zoo/releases) and drafting a release. When the time is there, we make a one line PR to change the version number ([example](https://github.com/larq/zoo/commit/cbe657d7a12b7872c078337270ae0c0e1750deed)).

Once we make the release I think we may want to go to 0.4 as it's a breaking change, but below 1.0.0 there are no real rules in general as far as I know.

Reply:
Correct. There is no need to change the version number here.

Reply:
reset to `0.3.0`